src/diffusers/loaders/lora.py

```diff
@@ -292,81 +223,78 @@ def lora_state_dict(
 
         return state_dict, network_alphas
 
-    @classmethod
-    def _best_guess_weight_name(
-        cls, pretrained_model_name_or_path_or_dict, file_extension=".safetensors", local_files_only=False
+    # This method acts as an interface to distinguish between `fuse_unet` and `fuse_transformer` arguments
+    # properly and lets us reuse the `fuse_lora` method of the superclass.
+    def fuse_lora(
```

Comment:
Explanation on why does this method exist in more details:
https://github.com/huggingface/diffusers/pull/8670#discussion_r1663362503